certified-connectors/Verified/apiProperties.json

```diff
@@ -0,0 +1,133 @@
+            "GetEnvelopeDescriptors",
+            "GetFlowsFlowIdJobsEnvelopeId",
+            "DeleteEnvelopesEnvelopeIdDocumentsDocumentId",
+            "GetEnvelopesEnvelopeIdDocumentsDocumentId",
+            "PostEnvelopeDescriptorsEnvelopeDescriptorIdEnvelopes",
+            "GetEnvelopesEnvelopeIdDocuments",
+            "PostEnvelopesEnvelopeIdDocuments",
+            "PostEnvelopesEnvelopeIdJobsGetSignLink",
+            "GetEnvelopesEnvelopeIdDocumentsDocumentIdFilesFileId",
+            "PostEnvelopeDescriptorsDefaultEnvelopes",
+            "GetEnvelopeDescriptorsDefault",
+            "PostEnvelopesEnvelopIdJobsSendNotification",
+            "PostEnvelopesEnvelopeIdDocumentsDocumentIdTemplatesTemplateIdUserData",
+            "GetEnvelopesEnvelopeIdDocumentsDocumentIdFiles",
+            "PostEnvelopesEnvelopeIdDocumentsDocumentIdStatusAborted"
+          ]
+        }
+      },
+      {
+        "templateId": "setheader",
+        "title": "Set company id for all requests",
+        "parameters": {
+          "x-ms-apimTemplateParameter.name": "x-namespace",
+          "x-ms-apimTemplateParameter.value": "/companies/@connectionParameters('companyid')",
+          "x-ms-apimTemplateParameter.existsAction": "override",
+          "x-ms-apimTemplate-policySection": "Request",
+          "x-ms-apimTemplate-operationName": [
+            "GetEnvelopesEnvelopeIdRecipients",
+            "PostEnvelopesEnvelopeIdRecipients",
+            "GetEnvelopesEnvelopeIdRecipientsRecipientId",
+            "PutEnvelopesEnvelopeIdRecipientsRecipientId",
+            "GetEnvelopesEnvelopeIdDocumentsDocumentIdFilesFileIdUrl",
+            "GetEnvelopesEnvelopeId",
+            "DeleteEnvelopesEnvelopeId",
+            "PutEnvelopesEnvelopeId",
+            "PutEnvelopesEnvelopeIdPublishStatus",
+            "PostEnvelopesEnvelopeIdDocumentsDocumentIdTemplatesTemplateIdUserData",
+            "PutEnvelopesEnvelopeIdAbortStatus",
+            "GetEnvelopesEnvelopeIdDocumentsDocumentIdFiles",
+            "PostEnvelopesEnvelopeIdDocumentsDocumentIdFiles",
+            "PostEnvelopesEnvelopeIdDocumentsDocumentIdStatusAborted",
+            "GetEnvelopeDescriptorsEnvelopeDescriptorId",
+            "PutEnvelopesEnvelopeIdTrashStatus",
+            "GetQueryEnvelopes",
+            "GetEnvelopeDescriptors",
+            "GetEnvelopesEnvelopeIdDocumentsDocumentId",
+            "DeleteEnvelopesEnvelopeIdDocumentsDocumentId",
+            "PostEnvelopeDescriptorsEnvelopeDescriptorIdEnvelopes",
+            "GetEnvelopesEnvelopeIdDocuments",
+            "PostEnvelopesEnvelopeIdDocuments",
+            "PostEnvelopesEnvelopeIdJobsGetSignLink",
+            "GetEnvelopesEnvelopeIdDocumentsDocumentIdFilesFileId",
+            "GetEnvelopeDescriptorsDefault",
+            "PostEnvelopeDescriptorsDefaultEnvelopes",
+            "PostEnvelopesEnvelopIdJobsSendNotification"
+          ]
+        }
+      }
+    ],
+    "publisher": "CRM Konsulterna i Sverige AB",
```

Comment:
@BenediktBergmann Can you please let us know why the publisher name is different from stackOwner name. Please start an email thread with us (ConnectorPartnerMgmtTeam@service.microsoft.com) and let us know how you are authorized to develop a connector "Verified AB" company.

Reply:
We (CRM Konsulterna) are the partner of Verified implementing the custom connector for them.
We already have talked with MS about it and Verified has approved to MS that we are the once building it for them.
I will check whether it was send to the mentioned email.

Reply:
We (CRM Konsulterna) are the partner of Verified implementing the custom connector for them.
We already have talked with MS about it and Verified has approved to MS that we are the once building it for them.
I will check whether it was send to the mentioned email.

Reply:
There is actually a conversation where the mentioned email address is included (Subject: Verified connector)
Verified wrote on 2022-02-15 4:30 AM that we (CRM-Konsulterna) have Verifieds permission to create and manage the certification process on Verified's behalf.
@Amjed-Ayoub wrote on 2022-02-16 at 12AM (CET) that it is okay and we should go forward.